.gitignore

```diff
@@ -60,3 +60,4 @@ meta/toolchain/gnu/local
 meta/toolchain/gnu/tarballs
 
 docs/brutal.pdf
+sources/libs/virtio/virtio-1.1-spec.pdf
```

Comment:
why this ?

Reply:
I have my own copy of the VirtIO spec in order to mark some useful things on it via Foxit Reader, but I guess I shouldn't push it. Maybe you want from me to remove it ?